backend/core/src/email/email.service.ts

```diff
@@ -108,7 +108,7 @@ export class EmailService {
     await this.send(
       email,
       jurisdiction.emailFromAddress,
-      "Bloom account access token",
+      "Partners Portal account access token",
```

Comment:
I believe we want these strings to be translated. If this ticket is on a timeline down to make a separate issue.